### ISSUE TITLE
there is a code defect need to fix

### ISSUE THREAD

**Issue by Light-Reflection** [apple/ml-cvnets#63]
In RANAGEAUGMENT model, if i set ml-cvnets\examples\range_augment\classificationresnet_50.yaml --->learn_augmentation.mode to 'none', then,  line 33 in cross_entropy_with_neural_aug.py will be error, because line 240 in base_cls.py set return data into 'x', and 'x' is a tensor, you need to add x into 'out_dict ' just like:

out_dict = {                
"augmented_tensor": None,
                "logits": x}

return out_dict.

then, it works

**Comment by farzadab** [comment]
Yes, if you disable `learn_augmentation`, you should also change both the loss to `cross_entropy` and `stats.checkpoint_metric` to `top1` (instead of `top1.logits`).
